.golangci.yml

```diff
@@ -0,0 +1,52 @@
+run:
+  tests: true
+  max-same-issues: 50
+  modules-download-mode: vendor
+
+output:
+  print-issued-lines: false
+
+linters:
+  enable-all: true
+  disable:
+    - maligned
+    - megacheck
+    - lll
+    - typecheck # `go build` catches this, and it doesn't currently work with Go 1.11 modules
+    - goimports # horrendously slow with go modules :(
+    - dupl # has never been actually useful
+    - gochecknoglobals
+    - gochecknoinits
+    - interfacer # author deprecated it because it provides bad suggestions
```

Comment:
This seems like a lot fo disabled linters. Would it be worth just having an enabled list?